src/plans/create/steps/other-settings/RootDeviceField.tsx

```diff
@@ -0,0 +1,79 @@
+import type { FC } from 'react';
+import { Controller } from 'react-hook-form';
+import { getRootDiskLabelByKey } from 'src/modules/Plans/views/details/components/SettingsSection/modals/EditRootDisk/getRootDiskLabelByKey';
+import { diskOptions } from 'src/plans/details/tabs/Details/components/SettingsSection/components/RootDisk/utils/constants';
+
+import { ExternalLink } from '@components/common/ExternalLink/ExternalLink';
+import { HelpIconPopover } from '@components/common/HelpIconPopover/HelpIconPopover';
+import Select from '@components/common/MtvSelect';
+import { FormGroup, SelectList, SelectOption, Stack, StackItem } from '@patternfly/react-core';
+import { ForkliftTrans, useForkliftTranslation } from '@utils/i18n';
+import { VIRT_V2V_HELP_LINK } from '@utils/links';
+
+import { useCreatePlanFormContext } from '../../hooks';
+
+import { otherFormFieldLabels, OtherSettingsFormFieldId } from './constants';
+
+const fieldId = OtherSettingsFormFieldId.RootDevice;
```

Comment:
why outside the component?

Reply:
By defining outside of the component it guarantees no re-initialization of these constants if the component goes through cycles of re-rendering.